testsuite/runtest.py

```diff
@@ -80,8 +91,8 @@
         os.symlink (os.path.join (test_source_dir, "src"), "./src")
     if not os.path.exists("./data") :
         os.symlink (test_source_dir, "./data")
-    if not os.path.exists("../common") :
-        os.symlink ("../../../testsuite/common", "../common")
+    if not os.path.exists(testroot + "/common") :
+        os.symlink (root + "/testsuite/common", testroot + "/common")
```

Comment:
This logic is failing for me.  The symlink ends up pointing to ../../../..../..//testsuite/common on my linux and macos machines.  
If I revert this line to the original, `os.symlink ("../../../testsuite/common", "../common")` the tests all run fine.